From 827b40098862b65fdc1a96288fbd930f16b3e49f Mon Sep 17 00:00:00 2001
From: Aurelien Jarno <aurelien@aurel32.net>
Date: Tue, 8 Mar 2016 15:16:40 +0100
Subject: [PATCH] Add sys/auxv.h wrapper to include/sys/

The GNU libc testsuite fails to build on powerpc/ppc64/ppc64le with the
following error:

    ../sysdeps/powerpc/test-get_hwcap.c:26:22: fatal error: sys/auxv.h: No such file or director

This is because test-get_hwcap.c includes <sys/auxv.h>, but we don't
provide a wrapper in include/sys. This patch adds one.

Changelog:
	* include/sys/auxv.h: New file.

(cherry picked from commit 0b8dedd38f304d796b6b9b349428bea7f1f7065f)
(cherry picked from commit 9d1399e89d431b34a5a6e5e48ee40721c22815b8)
---
 include/sys/auxv.h | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 include/sys/auxv.h

diff --git a/include/sys/auxv.h b/include/sys/auxv.h
new file mode 100644
index 0000000..dede2c3
--- /dev/null
+++ b/include/sys/auxv.h
@@ -0,0 +1 @@
+#include <misc/sys/auxv.h>
-- 
2.6.2

